docs/user/aws/limits.md

```diff
@@ -0,0 +1,74 @@
+than 5 clusters, you will need to increase this limit.
+
+## Elastic Network Interfaces (ENI)
+
+The default installation creates 21 + the number of availability zones of ENIs (e.g. us-east-1 = 21 + 6 = 27 ENIs). 
+The default limit per region is 350. Additional ENIs are created for additional machines and elastic load balancers 
+created by cluster usage and deployed workloads. A service limit increase here may be required to satisfy the needs of 
+additional clusters and deployed workloads.
+
+## Elastic IP (EIP)
+
+For a single, default cluster, your account will have the needed capacity limits required. There is one exception, 
+“EC2-VPC Elastic IPs”. The installer creates a public and private subnet for each availability zone within a region to 
+provision the cluster in a highly available configuration. In each private subnet, a separate NAT Gateway is created and 
+requires a separate elastic IP. The default limit of 5 is sufficient for most regions and a single cluster. For the 
+us-east-1 region, a higher limit is required. For multiple clusters, a higher limit is required. Please see this map for 
+a current region map with availability zone count. We recommend selecting regions with 3 or more availability zones.
+
+## NAT Gateway
+
+The default limit for NAT Gateways is 5 per availability zone. This is sufficient for up to 5 clusters in a dedicated 
+account. If you intend to create more than 5 clusters, you will need to request an increase to this limit.
+
+## VPC Gateway
+
+The default limit of VPC Gateways (for S3 access) is 20. Each cluster will create a single S3 gateway endpoint within 
+the new VPC. If you intend to create more than 20 clusters, you will need to request an increase to this limit.
+
+## Security Groups
+
+Each cluster creates 10 distinct security groups. The default limit of 2,500 for new accounts allows for many clusters 
+to be created.
+
+## Instance Limits
+
+By default, a cluster will create 6 nodes (3 masters and 3 workers). Currently, these are m4.large and within a new 
+account’s default limit. If you intend to start with a higher number of workers, enable autoscaling and large workloads 
+or a different instance type, please ensure you have the necessary remaining instance count within the instance type’s 
+limit to satisfy the need. If not, please open a limit increase with AWS to match the need.
+
+## Elastic Load Balancing (ELB/NLB)
+
+By default, each cluster will create 2 network load balancers for the master API server (1 internal, 1 external) and a 
+single classic elastic load balancer for the router. Additional Kubernetes LoadBalancer Service objects will create 
+additional load balancers.
+
+[AWS Load Balancing][load-balancing]
+
+## Example: Using N. Virginia (us-east-1)
+
+To use N. Virginia (us-east-1) for a new cluster, please submit a limit increase for VPC Elastic IPs similar to the 
```

Comment:
Wouldn't this belong under the EIP section?

Reply:
Moved it. I thought my example might have other increases required to hit a couple services. Moved directly under. Thanks!